### PR TITLE
snapshots ofESLint Airbnb voilations and HoundCI

### PR DESCRIPTION
**Pull request by andela-cofor**
#26 screenshots of ESLint Airbnb style violations in the terminal and HoundCI comment on a PR

#### What does this PR do?
PR checks for  ESLint Airbnb style violations in the terminal and HoundCI comment on a PR

#### Description of Task to be completed?
Eslint flags when rules are broken, HoundCI comments on a PR

#### How should this be manually tested?
* ```npm install ```
* ```eslint filename.js```

#### Any background context you want to provide?
N/A

#### Screenshots 
Hound CI
screenshot 41

ESLint 
screenshot 33

#### Questions:
N/A

### CODE REVIEW

**Comment by nelsonabienorin** [comment]
Do you mean eslint filrname.js instead of eslint filename.js ?

**Comment by andela-cofor** [comment]
Yes Nelson, edited it to filename.js

**Comment by victor-enogwe** [comment]
looks good

**Comment by nelsonabienorin** [comment]
Thats cool!

File: 06-building-a-program-start-to-finish/setting-up-linters/output/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#### Output
+- **Using**: Terminal & GitHub
+- **Deliver**: wo screenshots of ESLint Airbnb style violations in the terminal and HoundCI comment on a PR
```
Comment (q-ode) [inline 06-building-a-program-start-to-finish/setting-up-linters/output/README.md]: Typo!

**Review by q-ode — COMMENTED** [review]
I can't access the images from within this network.

It's blocked. Kindly host on a provider available to people on the Network.

**Comment by andela-cofor** [comment]
Is this better added screen shots ?

File: 06-building-a-program-start-to-finish/setting-up-linters/output/README.md (hunk 1 above)
Comment (andela-cofor) [reply 06-building-a-program-start-to-finish/setting-up-linters/output/README.md]: Noted...

Comment (andela-cofor) [reply 06-building-a-program-start-to-finish/setting-up-linters/output/README.md]: @q-ode Done...

**Comment by andela-cofor** [comment]
@q-ode typo has been corrected and pushed

**Pull request merged**